### DISCUSSION TITLE
Access to Account pages w/out signup

### DISCUSSION THREAD

**Discussion by ghost** [scosman/CMSaasStarter#90] [General]
First, I appreciate what you've done here. You've greatly helped me kick-start my project. Everything is laid out so elegantly and it's very well organized. 

I'm a Python developer learning web dev and I'm still ramping up with everything (svelte, tailwind, ...). With that said, how do I access (view) my account pages w/out going through auth, 'signing up'? 

I haven't yet wired the app to SuppaBase and waiting to do so after I get a few things accomplished. For now, I'd like to easily access all the 'gated' pages during dev.

Thanks!

**Comment by scosman** [comment]
You could hack that in, but quickly you'll just see bugs because those pages reply on account info. I'd suggest just setting up your environment and log in for realistic dec experience.

**Reply by ghost** [reply]
I'll do that. Appreciate the feedback.
